.circleci/config.yml

```diff
@@ -91,6 +91,9 @@ workflows:
       - hmpps/gradle_owasp_dependency_check:
           context:
             - hmpps-common-vars
+      - hmpps/trivy_latest_scan:
+          context:
+            - hmpps-common-vars
```

Comment:
would it make sense parameterising these so it doesn't just go to dps-security-alerts or whatever the default is?

Maybe we could remove some of the defaulting the orb does generally if there isn't a sensible default.

Reply:
Yes I did think that, but Kotlin projects dont have a shared slack channel like you have for typescript.  I'll add it in, like you say good to have it there.